users/tests/test_user.py

```diff
@@ -72,3 +72,11 @@ def test_no_authenticated_client(client, django_user_model):
     response = client.get('/profile/')
     assert response.status_code == 302
     assert response.url == '/login/?next=/profile/'
+
+
+@pytest.mark.django_db
+def test_update_user(user):
```

Comment:
You already have:
def create_user(db, django_user_model, test_password):
Consider support of calling with fixture test_password or not

Reply:
Any response on this?

Reply:
> Any response on this?

OK i see it was  changed